torchinductor/compile_fx.py

```diff
@@ -72,6 +73,9 @@ def compile_fx_inner(
     num_fixed=0,
     is_backward=False,
 ):
+    if count_calls(gm.graph) == 0:
```

Comment:
surprised we get here - iirc dynamo raises `raise exc.SkipFrame()` if `count_calls` returns 0. Did you see this downstream of dynamo?

Reply:
Yeah, I am surprised too. This is DDP magic. Need to look into more details whats going on.

Reply:
Can you dump the graph? Perhaps its a graph of constants? Or something funky like an empty gm?

Reply:
Could be from a backwards graph...